docs/model-dev-guide/debug-models.rst

```diff
@@ -117,9 +117,10 @@ This step assumes you have a working local environment for training. If you do n
    If your per-method checks in :ref:`Step 2 <step2>` passed but local test mode fails, your
    ``Trial`` subclass might not be implemented correctly. Double-check the documentation. It is also
    possible that you have found a bug or an invalid assumption in the Determined software and should
-   `file a GitHub issue <https://github.com/determined-ai/determined/issues/new>`__ or contact
```

Comment:
This link definitely still works. It just requires a user to login. Let's keep this and I'll add it to the ignore list for link checking.

Reply:
Better yet, just point to `https://github.com/determined-ai/determined/issues`

Reply:
done